### PR TITLE
Fix: View Created for Contact Us Page(#563)

### PR DESCRIPTION
**Pull request by Ram81**
@deshraj please review.
contact-us

### CODE REVIEW

**Comment by deshraj** [comment]
@aka-jain any changes that you want in the UI for this?

File: frontend/src/views/web/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        				</div>
+        			<!-- Email -->
+        				<div class="input-field align-left">
+            				<input type="text" id="email" name="email" required>
+            				<span class="form-icon"><i class="fa fa-envelope"></i></span>
+            				<label for="email">Email *</label>
+        				</div>
+        			<!-- Message -->
+        				<div class="input-field align-left">
+            				<input type="text" id="message" name="message" required>
```
Comment (spyshiv) [inline frontend/src/views/web/contact-us.html]: @Ram81 : use `<textarea></textarea>` for 'message' instead of `<input>` since details shouldn't be inline.

Comment (Ram81) [inline frontend/src/views/web/contact-us.html]: @spyshiv sure

**Comment by aka-jain** [comment]
@deshraj No, not for now. Will change design once I design new input fileds.

File: frontend/src/js/route-config/route-config.js [hunk 2]
```diff
@@ -307,12 +307,20 @@
...
+        
         var our_team = {
             name: 'our-team',
             url: "/team",
             templateUrl: baseUrl + "/web/our-team.html",
             title: "Our Team"
         };
+        
+        var contact_us = {
+        	name: 'contact-us',
```
Comment (aka-jain) [inline frontend/src/js/route-config/route-config.js]: @Ram81 please make sure to make code properly indented :)

Comment (Ram81) [inline frontend/src/js/route-config/route-config.js]: @aka-jain sorry for this I will change it right now

**Comment by Ram81** [comment]
@aka-jain Done

File: frontend/src/views/web/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<div class="white-bg view-container">
+    <main-header></main-header>
+    
+    <div class="grad-container">
+        <div class="row">
+            <div class="col s12 m8 l8">
+                    <br><br>
+                    <h4 align="center" class="text-dark-black"><strong>Contact Us</strong></h4>
+                    <form name="contactusForm" ng-submit="" id="contact-us" novalidate>
+        			<!-- Name -->
```
Comment (deshraj) [inline frontend/src/views/web/contact-us.html]: @Ram81 looks like the code between line 10-38 is not indented properly. Please use 4 space indentation.

**Comment by Ram81** [comment]
@deshraj please review

**Comment by RishabhJain2018** [comment]
@Ram81 Please solve the merge conflicts.

**Comment by Ram81** [comment]
@RishabhJain2018 Will do right now

**Pull request closed without merging**